### ISSUE TITLE
Do not display spectators in the statsboard

### ISSUE THREAD

**Issue by Sonix-** [teeworlds/teeworlds#2150]
Spectators shouldn't be shown in the statsboard instead display only stats of players that are ingame.

![screenshot_2019-06-13_22-58-21](https://user-images.githubusercontent.com/2030615/59547727-b85a0980-8f43-11e9-9862-8195c8a6eb8f.png)

**Comment by Dune-jr** [comment]
@Ryozuki any thoughts why we should keep the spectators there?

**Comment by Ryozuki** [comment]
> @Ryozuki any thoughts why we should keep the spectators there?

Is it possible that a spectator played during the match but not the whole match and maybe he wants to see his stats? If this doesn't make sense you can just ignore it then.

**Comment by Sonix-** [comment]
Im not talking about removing spec stats here, if he joins back the stats would still be there.. theres no sense in displaying someones stats in a game hes not even playing

**Comment by ChillerDragon** [comment]
On a random public server where people join and leave and chill in spec this could possible be interesting. But in real games it probably mostly pollutes the statsboard with spectators.

**Comment by sepulzera** [comment]
+1 for removing spectators. Eventually always display the stats for the current player even while spectating. How would the statsboard look like with 64 players? Gratuitous bloated i guess.

**Comment by oy** [comment]
Yeah, showing your own stats in case you're spectator and remove all others like @sepulzera suggested is a good compromise.
For 64 players it probably works as the scoreboard, showing only important/close ones.

**Comment by Sonix-** [comment]
My bad, apparently it's fixed in the next version by #2128 
Should I leave it open in case you want to discuss about showing personal stats even if in spectators?

**Comment by Dune-jr** [comment]
That's probably a good way to go about it, yeah.
